bundle/regal/rules/idiomatic/prefer_strings_count.rego

```diff
@@ -0,0 +1,35 @@
+# METADATA
+# description: Prefer to use `strings.count`
+package regal.rules.idiomatic["prefer-strings-count"]
+
+import rego.v1
+
+import data.regal.ast
+import data.regal.capabilities
+import data.regal.result
+
+# METADATA
+# description: Missing capability for built-in function `strings.count`
+# custom:
+#   severity: warning
+notices contains result.notice(rego.metadata.chain()) if not capabilities.has_object_keys
+
+# METADATA
+# description: flag calls to `count`` where the first argument is a call to `indexof_n`
+report contains violation if {
```

Comment:
Is it too much effort to catch cases like this?

```
indexes := indexof_n(foo)
count := count(indexes) 
```

Reply:
I suppose then we'd need to check if indexes is used elsewhere too...

Reply:
Yeah, that's too much effort IMHO :D I'd be surprised if this was a violation in more than 1% of the policy repos out there already... but if we find it being "popular" later, we can work on improving it.